test/uk/gov/hmrc/catalogue/github/GithubV3TeamsRepositoryDataSourceSpec.scala

```diff
@@ -61,14 +68,13 @@ class GithubV3TeamsRepositoryDataSourceSpec extends GithubWireMockSpec with Scal
           GhTeam("D", 4) -> List(GhRepository("D_r", 4, "url_D")))))
 
       dataSource.getTeamRepoMapping.futureValue shouldBe List(
-        Team("A", List(Repository("A_r2", "url_A2"))),
-        Team("C", List()),
-        Team("D", List(Repository("D_r", "url_D"))))
+        TeamRepositories("A", List(Repository("A_r2", "url_A2", isInternal = true))),
+        TeamRepositories("C", List()),
+        TeamRepositories("D", List(Repository("D_r", "url_D", isInternal = true))))
 
     }
 
-    "Filter out teams according to the hidden config" in new WithApplication(
-      FakeApplication(additionalConfiguration = Map("github.hidden.teams" -> testHiddenTeams.mkString(",")))) {
+    "Filter out teams according to the hidden config" in {
```

Comment:
Is there a reason we're no longer testing that the config is parsed correctly? These tests are not as robust as they were previously